src/pages/test-report/components/competency/competency.constants.ts

```diff
@@ -0,0 +1,42 @@
+
+export const competencyLabels = {
```

Comment:
can this be an `enum` too?

Reply:
Oh perhaps not as it needs to be used in the template?

Reply:
Done